packages/contracts/hardhat.config.js

```diff
@@ -29,35 +29,35 @@ module.exports = {
   paths: {
     artifacts: "./build",
   },
-  networks: {
-    sokol: {
-      url: `https://sokol.poa.network`,
-      accounts: [`0x${PRIVATE_KEY}`],
-      gasPrice: 1000000000,
-    },
-    xdai: {
-      url: `https://rpc.xdaichain.com`,
-      accounts: [`0x${PRIVATE_KEY}`],
-      gasPrice: 1000000000,
-    },
-    kovan: {
-      url: `https://kovan.infura.io/v3/${INFURA_PROJECT_ID}`,
-      accounts: [`0x${PRIVATE_KEY}`],
-    },
-    rinkeby: {
-      url: `https://rinkeby.infura.io/v3/${INFURA_PROJECT_ID}`,
-      accounts: [`0x${PRIVATE_KEY}`],
-    },
-    ganache: {
-      url: "http://127.0.0.1:8555",
-      defaultBalanceEther: 1000,
-    },
-    mainnet: {
-      url: `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`,
-      accounts: [`0x${PRIVATE_KEY}`],
-      gasPrice: 4500000000,
-    },
-  },
+  // networks: {
+  //   sokol: {
+  //     url: `https://sokol.poa.network`,
+  //     accounts: [`0x${PRIVATE_KEY}`],
+  //     gasPrice: 1000000000,
+  //   },
+  //   xdai: {
+  //     url: `https://rpc.xdaichain.com`,
+  //     accounts: [`0x${PRIVATE_KEY}`],
+  //     gasPrice: 1000000000,
+  //   },
+  //   kovan: {
+  //     url: `https://kovan.infura.io/v3/${INFURA_PROJECT_ID}`,
+  //     accounts: [`0x${PRIVATE_KEY}`],
+  //   },
+  //   rinkeby: {
+  //     url: `https://rinkeby.infura.io/v3/${INFURA_PROJECT_ID}`,
+  //     accounts: [`0x${PRIVATE_KEY}`],
+  //   },
+  //   ganache: {
+  //     url: "http://127.0.0.1:8555",
+  //     defaultBalanceEther: 1000,
+  //   },
+  //   mainnet: {
+  //     url: `https://mainnet.infura.io/v3/${INFURA_PROJECT_ID}`,
+  //     accounts: [`0x${PRIVATE_KEY}`],
+  //     gasPrice: 4500000000,
+  //   },
+  // },
```

Comment:
Is hardhat not working correctly, or why is this commented out?

Reply:
I comment it out to test the contracts locally. I just forgot to uncomment them afterwards